crates/evm/src/evm/handler.rs

```diff
@@ -485,7 +485,29 @@ fn calc_diff_size<EXT, SPEC: Spec, DB: Database>(
                 // So we need to only force the nonce change.
                 account.account_info_changed = true;
             }
-            JournalEntry::AccountDestroyed { address, .. } => {
+            JournalEntry::AccountDestroyed {
+                address,
+                target,
+                was_destroyed,
+                had_balance,
+            } => {
+                // This event is produced only if acc.is_created() || !is_cancun_enabled
+                // State is not changed:
+                // * if we are after Cancun upgrade and
```

Comment:
```suggestion
                // * if  Cancun fork is activated and
```

Reply:
https://github.com/bluealloy/revm/blob/bbc8d81dbe2a6a4d184e32fa540e1c4e248a65c9/crates/revm/src/journaled_state.rs#L530

Reply:
Pasting the text as-is doesn't mean that the wording is correct.

Reply:
resolving this as both wording work